File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        bool dup = false;
        for(int i = 0; i < size(nums);i++){
            for (int j = 1; j < (size(nums) - i); j++){
                if (nums[i] == nums[j + i]){
                    return true;
                    dup = true;
                } 
            }
        }
        if (!dup) return false;
    }
};

File: Data Structures & Algorithms/sliding-window-maximum/submission-0.py
class Solution:
    def findMax(self, nums, l, r):
        return max(nums[l:r+1])
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        l = 0
        r = k - 1
        output = []
        while r <= len(nums) - 1:
            output.append(self.findMax(nums,l,r))
            l += 1
            r += 1
        return output